src/librustc_mir/interpret/terminator.rs

```diff
@@ -254,107 +264,110 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 // We want to pop this frame again in case there was an error, to put
                 // the blame in the right location.  Until the 2018 edition is used in
                 // the compiler, we have to do this with an immediately invoked function.
-                let res =
```

Comment:
rustfmt made a mess here :(
Hiding whitespace changes helps a bit.

Reply:
I guess since we are on 2018 by now we could use a try block now -- if you're fine with that I can do that in this PR.

Reply:
Yea, a try block is fine by me